Source/Core/Core/Brawlback/BrawlbackUtility.h

```diff
@@ -327,78 +327,12 @@ namespace Brawlback {
     };
     namespace Match
     {   
-        struct PlayerFrameDataImpl {
-            PlayerFrameData _playerFrameData;
-
-            // do these impact the size of the struct?
-            // wouldn't the vtable ptr screw with it being interpreted on gameside???
-            // (since the gameside structs don't have these ctors)
-            PlayerFrameDataImpl() {
-                _playerFrameData.frame = 0;
-                _playerFrameData.playerIdx = 0;
-                _playerFrameData.pad = BrawlbackPadImpl()._brawlbackPad;
-            }
-            PlayerFrameDataImpl(u32 _frame, u8 _playerIdx)
-            {
-                _playerFrameData.frame = _frame;
-                _playerFrameData.playerIdx = _playerIdx;
-                _playerFrameData.pad = BrawlbackPadImpl()._brawlbackPad;
-            }
-        };
-
-        struct FrameDataImpl {
-            FrameData _frameData;
-
-            FrameDataImpl()
-            {
-                _frameData.randomSeed = 0;
-                for (int i = 0; i < MAX_NUM_PLAYERS; i++) {
-                    _frameData.playerFrameDatas[i] = PlayerFrameDataImpl()._playerFrameData;
-                }
-            }
-            FrameDataImpl(u32 frame)
-            {
-                _frameData.randomSeed = 0;
-                for (u8 i = 0; i < MAX_NUM_PLAYERS; i++) {
-                    _frameData.playerFrameDatas[i] = PlayerFrameDataImpl(frame, i)._playerFrameData;
-                }
-            }
-        };
-        struct PlayerSettingsImpl
-        {
-            PlayerSettings _playerSettings;
-        };
-
-        struct GameSettingsImpl
-        {
-            GameSettings _gameSettings;
-        };
         struct Game {
```

Comment:
Maybe my visual studio search skills are terrible.. but is this struct Game actually used anywhere? When I comment it out, the project still builds fine